### PR TITLE
Select payment gateway when it is the only option

### PR DESCRIPTION
**Pull request by ryanmitchell**

### CODE REVIEW

**Comment by sampoyigi** [comment]
Not sure this is needed... Try setting the enabled payment as the default and it gets pre-selected on the checkout page.

**Comment by ryanmitchell** [comment]
But you can only have one payment as default and on multiple locations they may have different options available.

**Comment by sampoyigi** [comment]
Ahhh.. got it... PR is for when more than one payment gateway is enabled site-wide but only one enabled location-wide and not the default..

**Comment by ryanmitchell** [comment]
Yep :)

**Comment by sampoyigi** [comment]
On second thought, make the changes in `OrderManager::getCurrentPaymentCode` to return the code of the first available payment when no session and default is available.

File: classes/OrderManager.php [hunk 1]
```diff
@@ -381,7 +381,7 @@ public function setCurrentPaymentCode($code)
 
     public function getCurrentPaymentCode()
     {
-        return $this->getSession('paymentCode') ?: optional($this->getDefaultPayment())->code;
+        return $this->getSession('paymentCode') ?: optional($this->getDefaultPayment() ?: $this->getPaymentGateways()->first())->code;
```
Comment (sampoyigi) [inline classes/OrderManager.php:384]: Yummy! 😉

**Review by sampoyigi — APPROVED** [review]

**Comment by ryanmitchell** [comment]
I’m getting the hang of this laravel stuff :)

**Pull request merged**